autogluon/buildspec-0-4.yml

```diff
@@ -0,0 +1,74 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK autogluon
+version: &VERSION 0.4.0
```

Comment:
Please make this the 0.4.2 version of the buildspec.yml file. Is there any reason we would want to continue to have AG 0.4.0 images once we start to publish AG 0.4.2?

Reply:
Good point, no we want `0.4.2` to be the new `0.4`. Will update.